api/output_policy.go

```diff
@@ -88,96 +82,22 @@ func (d *OutputPolicyError) buildSamplePolicy() (string, error) {
 }
 
 // Determine whether the given path requires the sudo capability
-func isSudoPath(client *Client, path string) (bool, error) {
-	sudoPaths, err := getSudoPaths(client)
-	if err != nil {
-		return false, fmt.Errorf("unable to retrieve list of paths that require sudo capability: %v", err)
-	}
-	if sudoPaths == nil || len(sudoPaths) < 1 {
-		// OpenAPI spec did not return any paths that require sudo,
-		// but the user probably still shouldn't see an error.
-		return false, nil
-	}
-
-	// Return early if the path is clearly one of the sudo paths.
+func isSudoPath(path string) bool {
+	// Return early if the path is any of the non-templated sudo paths.
 	if _, ok := sudoPaths[path]; ok {
-		return true, nil
+		return true
 	}
 
 	// Some sudo paths have templated fields in them.
 	// (e.g. sys/revoke-prefix/{prefix})
-	// The keys in the sudoPaths map are actually regular expressions,
+	// The values in the sudoPaths map are actually regular expressions,
 	// so we can check if our path matches against them.
-	for sudoPath := range sudoPaths {
-		r, err := regexp.Compile(fmt.Sprintf("^%s$", sudoPath))
-		if err != nil {
-			continue
-		}
-
-		match := r.Match([]byte(fmt.Sprintf("/%s", path))) // the OpenAPI response has a / in front of each path
+	for _, sudoPathRegexp := range sudoPaths {
+		match := sudoPathRegexp.Match([]byte(fmt.Sprintf("/%s", path))) // the OpenAPI response has a / in front of each path
```

Comment:
```suggestion
		match := sudoPathRegexp.MatchString(fmt.Sprintf("/%s", path)) // the OpenAPI response has a / in front of each path
```

Reply:
It also seems like the "/" is removed first before the call to `isSudoPath` and then added back here. Maybe it would be simpler to keep it in the original?